### ISSUE TITLE
(Feature) New possibility of sending a post

### ISSUE THREAD

**Issue by dcapeletti** [diaspora/diaspora#7174]
While I am reading the Stream in the middle of my reading would send a new publication without losing sight of what I'm reading. It would be interesting to consider adding a button to display the pop-publications editor at the top, near the head. When the publication is sent, the editor hides.

**Comment by Flaburgan** [comment]
Honestly, I feel that this feature would add complexity for something pretty useless. The user can simply middle click the "Stream" link in the header to open a new tab, publish the new post and then close that tab. Or am I missing something? If not, I would be in favor of closing this one.

**Comment by SuperTux88** [comment]
I don't think this feature would add a lot of complexity and while this is really only nice-to-have (because you can do it now as you describe it, and simply write the post in a new tab), we could add this in the form of a icon next to the notification/message icon like in the mobile/responsive view (which we already have) which appears when you start scrolling down (or is always visible). So if somebody wants to add this feature, I wouldn't be against it, and if nobody wants to do it, it doesn't matter (so if others thinks this should be closed, I don't care) :)

**Comment by Waithamai** [comment]
I don't consider this useless. It's not a must-have but it would be nice to have. Maybe it could be done similar to what happens when writing a private message by clicking on the icon on a user's profile. A message window opens as overlay to the profile page. But, after sending a new post, the window should just be hidden again, not forward to a reloaded stream.
